releases/pims/0.2.2/pims/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo $SP_DIR
+
+
+$PYTHON setup.py install --single-version-externally-managed --root /
```

Comment:
Hmm. I forget. It's possible I copied this from an arbitrary recipe on my first day using conda.